src/Gateway.php

```diff
@@ -116,6 +124,56 @@ public function payment_fields() {
 		);
 	}
 
+	/**
+	 * Add additional billing fields to the checkout form.
+	 *
+	 * @param array $fields The billing fields.
+	 * @return array
+	 */
+	public function additional_billing_fields( $fields ) {
+
+		// NOTE: If you'll be changing the property name, make sure to update it in the JavaScript code too (file: assets/js/ledyer-payments.js).
+		$fields['billing_customer_reference1'] = array(
+			'label'       => __( 'Reference 1', 'ledyer-payments-for-woocommerce' ),
```

Comment:
Should the label actually be reference 1 and 2? I'm not sure if that is descriptive enough for what this is